src/tr/eldermanga/src/eu/kanade/tachiyomi/extension/tr/eldermanga/ElderManga.kt

```diff
@@ -0,0 +1,237 @@
+
+    override val lang = "tr"
+
+    override val supportsLatest = true
+
+    override val versionId = 1
+
+    override val client = network.cloudflareClient.newBuilder()
+        .rateLimit(3)
+        .build()
+
+    private val json: Json by injectLazy()
+
+    override fun headersBuilder() = super.headersBuilder()
+        .set("Referer", "$baseUrl/")
+
+    override fun popularMangaRequest(page: Int): Request =
+        GET("$baseUrl/search?page=$page&search=&order=4", headers)
+
+    override fun popularMangaNextPageSelector() =
+        "section[aria-label='navigation'] li:has(a[class~='!text-gray-800']) + li > a:not([href~='#'])"
+
+    override fun popularMangaSelector() = "section[aria-label='series area'] .card"
+
+    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
+        title = element.selectFirst("h2")!!.text()
+        thumbnail_url = element.selectFirst("img")?.absUrl("src")
+        setUrlWithoutDomain(element.selectFirst("a")!!.absUrl("href"))
+    }
+
+    override fun latestUpdatesRequest(page: Int) =
+        GET(baseUrl, headers)
+
+    override fun latestUpdatesNextPageSelector() = null
+
+    override fun latestUpdatesSelector() = throw UnsupportedOperationException("Not used")
+
+    override fun latestUpdatesFromElement(element: Element) =
+        throw UnsupportedOperationException("Not used")
+
+    override fun fetchLatestUpdates(page: Int): Observable<MangasPage> {
+        return client.newCall(latestUpdatesRequest(page))
+            .asObservableSuccess()
+            .map { response ->
+                latestUpdatesParse(response)
+            }
+    }
+
+    override fun latestUpdatesParse(response: Response): MangasPage {
+        val document = response.asJsoup()
+
+        // Ana sayfadaki grid içinde bulunan manga kartlarını seçiyoruz
+        val elements =
+            document.select("div.grid.grid-cols-1.lg\\:grid-cols-2.xl\\:grid-cols-3 > div")
+
+        val mangas = mutableListOf<SManga>()
+
+        for (i in 0 until elements.size) {
+            try {
+                val element = elements[i]
+
+                // Manga başlığı
+                val titleElement = element.select("h2")
+                if (titleElement.isEmpty()) continue
+                val title = titleElement[0].text()
+
+                // Manga URL'si
+                val linkElements = element.select("a")
+                if (linkElements.isEmpty()) continue
+                val mangaUrl = linkElements[0].attr("href")
+                if (!mangaUrl.contains("/manga/")) continue
+
+                // Manga kapak resmi
+                val imgElements = element.select("div.card-image img")
+                val thumbnail = if (imgElements.isNotEmpty()) imgElements[0].attr("src") else ""
+
+                // En son bölüm bilgisi
+                val chapterElements = element.select("div.flex.pt-0.flex-col.gap-2\\.5 > a")
+                val hasNewChapter = chapterElements.isNotEmpty()
```

Comment:
use `selectFirst` which is nullable to see if something is missing instead of `isEmpty()`